cmd/snap-bootstrap/degraded-recover-mode.svg

```diff
@@ -1,3 +1,3 @@
R/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 885px; margin-left: 397px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 11px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #ffffff; white-space: nowrap; ">not found</div></div></div></foreignObject><text x="397" y="888" fill="#000000" font-family="Helvetica" font-size="11px" text-anchor="middle">not found</text></switch></g><rect x="312" y="665" width="170" height="65" rx="9.75" ry="9.75" fill="#ffffff" stroke="#000000" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 168px; height: 1px; padding-top: 698px; margin-left: 313px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">open unencrypted save</div></div></div></foreignObject><text x="397" y="701" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">open unencrypted save</text></switch></g><rect x="482" y="60" width="190" height="70" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 188px; height: 1px; padding-top: 95px; margin-left: 483px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; font-weight: bold; white-space: normal; word-wrap: normal; ">NOTE: state names correspond to function names but without whitespace between words</div></div></div></foreignObject><text x="577" y="99" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle" font-weight="bold">NOTE: state names correspond to...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
it's not a big deal, but it helps me to have all the states labeled with s1, s2, etc. so that I can run some static analysis on the state diagram to ensure we don't have loops etc and to calculate what the full set of paths through the state diagram is, and it's easier to use s4 as a node label than it is to use unlockDataRunKey for example, but certainly don't feel like you need to change it, I can handle this in a followup if you prefer